app/src/main/java/com/sun/premierleague/ui/splash/SplashActivity.kt

```diff
@@ -0,0 +1,24 @@
+package com.sun.premierleague.ui.splash
+
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import androidx.appcompat.app.AppCompatActivity
+import com.sun.premierleague.MainActivity
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        Looper.myLooper()?.let {
+            Handler(it).postDelayed({
+                startActivity(MainActivity.getIntent(this))
+                finish()
+            }, DELAY_MILLIS)
+        }
```

Comment:
Sao phải delay thế 
https://android.jlelse.eu/right-way-to-create-splash-screen-on-android-e7f1709ba154